.expeditor/verify_private.pipeline.yml

```diff
@@ -412,7 +412,7 @@ steps:
   #
   - label: ":cypress: (flaky included) IAMv1"
     command:
-      - IAM=v1.0 integration/run_test integration/tests/cypress_e2e.sh
+      - IAM=v1 integration/run_test integration/tests/cypress_e2e.sh
```

Comment:
must be `v1 | v2 | v2.1` to match the way the acceptance/dev builds are tagged in terraform (you can find the tags on the `.tf` files in the [terraform directory](https://github.com/chef/automate/blob/master/terraform/test-environments/scenarios))